src/Symfony/Bundle/SecurityBundle/Tests/Functional/HttpBasicLdapTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SecurityBundle\Tests\Functional;
+
+use Symfony\Component\HttpKernel\Kernel;
+
+class HttpBasicLdapTest extends WebTestCase
+{
+    public function testKernelBoot()
+    {
+        $kernel = self::createKernel(array('test_case' => 'HttpBasicLdap', 'root_config' => 'config.yml'));
```

Comment:
shouldn't we use the short syntax array? 🤔

Reply:
@pierre-tranchard it does not pass the symfony linter, look at others sources 👎

Reply:
@pierre-tranchard the short syntax is not allowed in symfony